course/assignments/R-intro-lectures-assignments.md

```diff
@@ -0,0 +1,60 @@
+# Week 1 (4 marks) Intro to programming
+1. Why is it beneficial to use a programming language rather than a spreadsheet software for data analyses?
+2. Get setup with Rstudio at home (or a lab computer after hours). Install R, Rstudio, and the necessary packages (tidyverse, rmarkdown). Use `install.packages(<package_name>)`. Load all the libraries you installed into your environment with `library(<package_name>)` and send us the output of running `sessionInfo()`.
```

Comment:
Reworded. Do you think it would be more useful for them to learn how to make a pdf? Sicne Rmd and md are so close in format anyways. I guess I can just go through that in the first lesson, should be fast.